app/lib/sanitize_config.rb

```diff
@@ -19,6 +19,11 @@ module Config
       node['class'] = class_list.join(' ')
     end
 
+    UNSUPPORTED_ELEMENTS_TRANSFORMER = lambda do |env|
+      return unless %w(h1 h2 h3 h4 h5 h6 blockquote pre li).include?(env[:node_name])
```

Comment:
fwiw if this line does what i think it does, then pre should be converted instead of maintained (unless you add css along the lines of `overflow-x: auto` or `white-space: pre-wrap`.

headers, blockquote, and li are reset by basics.scss regardless. for that matter, ol and ul are also safely reset by the current css.